docs/UserGuide.adoc

```diff
@@ -342,20 +388,21 @@ Card must be selected first before answering.
 * `answer c/0` Again +
 I don't know the answer to this card. Card will then be requeued into the same session again.
 * `answer c/1` Passable +
-I kind of know the answer, but need more practice. Card will then be scheduled for the near future. To get more practice.
+I think I know the answer, but need more practice. Card will then be scheduled for the near future. To get more practice.
 * `answer c/2` Easy A+ +
 Card will then be scheduled further into the future. Other cards need more review than this one.
 
-[TIP]
+[NOTE]
 ====
 This gives feedback to the scheduler algorithm to automatically schedule the card for review at a time when it thinks you are likely to forget it.
 
 This design is based off the well known forgetting curve, you can read more about it here https://en.wikipedia.org/wiki/Forgetting_curve
 ====
 
+[[schedule]]
 ==== Show flashcards for review: `Schedule`
-Using the `answer` command to answer a flash card will automatically schedule the card based on our spaced repetition algorithm.
-However sometimes when you need to manually `schedule` a card for review you can do so as well.
+Using the `xref:answer[answer]` command to answer a flash card will automatically schedule the card based on our spaced repetition algorithm.
```

Comment:
we've been using flashcard without spaces. So best to remove the space between flash and card in this line too

Reply:
okay